.eslintrc

```diff
@@ -1,15 +0,0 @@
-{
-  "extends": "./node_modules/@condenast/cn-js-styleguide/.eslintrc",
-  "env": {
-    "node": true,
-    "mocha": true,
-    "es6": true
-  },
-  "rules": {
-    "strict": [0, "global"],
-    "global-strict": [0, "always"],
-    "indent": [1, 2],
-    "consistent-return": 0,
```

Comment:
interesting that this rule was turned off — consistent return is important to allow for some potential V8 optimization